submissions/OwoeyeGbenga/landing_page/index.html

```diff
@@ -0,0 +1,149 @@
+<!doctype html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>Pie in the Sky Bakery, Springdale, VT</title>
+	<link href='http://fonts.googleapis.com/css?family=Cabin|Niconne' rel='stylesheet' type='text/css'>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="css/style.css" rel="stylesheet" type="text/css">
+</head>
+<body>
+<div class="wrapper">
+	<header class="row">
+		<a href="/"><img src="img/glitz.png" alt="Glitz logo"></a>
+	</header>
+	<nav class="row">
+		<ul class="full-width"> 
+            <li><a href="#">Contact</a></li>
+            <li><input type="text" placeholder="Search.." style="width: 200px;"></li>
+			<li><a href="signup.html">Register</a></li>
+			<li><a href="login.html">Login</a></li>
+			<li><a href="#"><img src="img/cart.png" alt="cart"></a></li>
+			<li><a href="#">WishList</a></li>
+		</ul>
+	</nav>
+	<div class="heading">
+        <div class="w3-content w3-section">
+            <img class="mySlides" src="img/slide1.jpg" style="width:100%">
+            <img class="mySlides" src="img/slide2.jpg" style="width:100%">
+            <img class="mySlides" src="img/slide3.jpg" style="width:100%">
+            <img class="mySlides" src="img/slide4.jpg" style="width:100%">
+            <img class="mySlides" src="img/slide5.jpg" style="width:100%">
```

Comment:
Since all these _img_ tags have the same inline style, you should consider adding the style to the common css class they share to avoid repetition

Reply:
done